test/colon/event-test.js

```diff
@@ -591,4 +591,21 @@ testHelpers.makeTests("can-stache-bindings - colon - event", function(name, doc,
 
 		QUnit.equal(map.get("clickCount"), 1, "function got called");
 	});
+
+	testIfRealDocument("on:click:value:to on button (#484)", function() {
```

Comment:
I read your [commit message](https://github.com/canjs/can-stache-bindings/pull/507/commits/b97ede71c78110c72d9d92df1041920c8fedf365) but I don’t quite understand, why does this test use `testIfRealDocument`?

Reply:
The `button` element in `vdom` doesn't contain `value` property. I spoke with @phillipskevin and @matthewp and they suggested to not run the test when in `vdom`.

Reply:
Did you figure out what the purpose of the value property on a button is?

Reply:
@matthewp According to Mozilla docs here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

Value:
> The initial value of the button. It defines the value associated with the button which is submitted with the form data. This value is passed to the server in params when the form is submitted.